app/controllers/concerns/scim_rails/response.rb

```diff
@@ -24,6 +24,23 @@ def json_scim_response(object:, status: :ok, counts: nil)
       end
     end
 
+    # TODO: Cut down on redundant code in this file
+
+    def json_scim_group_response(object:, status: :ok, counts: nil)
+      case params[:action]
+      when "index"
+        render \
```

Comment:
🌵 🌵 the usage of `\` here is odd

Reply:
This was something I moved, it was in the original code